File: Web.Javascript/services/userService.js
app.service('userService', function ($http) {

    var getUsers = function() {
        return $http.get('http://localhost:54027/User/GetUsers').then(function (response) {
            return response.data;
        });
    }

    var sortUsers = function(users) {
        return users.concat().sort();
    }

    return {
        message: 'Hello from angular service!',
        getUsers: getUsers,
        sortUsers: sortUsers
    };
})

File: Web.Net/Services/UserService.cs
using System.IO;
using System.Linq;

namespace Web.Net.Services
{
    public class UserService : IUserService
    {
        public string[] GetUsers()
        {
            var users = File.ReadAllLines(@"C:\JSUnitTestDemo\WEb.Net\usersSource.txt");
            return users;
        }

        public string[] SortUsers(string[] users)
        {
            return users.OrderBy(x => x).ToArray();
        }

        public string Message
        {
            get { return "Hello from .NET service!"; }
        }
    }
}